### PR TITLE
Update to recent version of several dependencies

### PR DESCRIPTION
**Pull request by dcoutts**
ledger specs, ledger, network, shell

Minor code update to the ToObject MuxTrace instance

### CODE REVIEW

**Comment by dcoutts** [comment]
This includes the necessary changes from #762

File: cabal.project [hunk 1]
```diff
@@ -59,64 +59,64 @@ source-repository-package
...
+  tag: 1b0d9ff9c7e201f0877ccec2a992c58e2b19c5ec
+  --sha256: 05vlmhxqsjn2y8ff864mm0jjbkl9pmb5qscbhwk6nrngqvxg00ys
   subdir: crypto
 
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-ledger
-  tag: 53ca3542019824d169378fa36a842eb4659b7ceb
-  --sha256: 1mwqcqfnxy14hs8g7x3q0ws3bhsmfhxpv8h7xf6r466n246fi83k
+  tag: 1b0d9ff9c7e201f0877ccec2a992c58e2b19c5ec
```
Comment (kevinhammond) [inline cabal.project:76]: Go for it

**Comment by dcoutts** [comment]
bors merge

**Comment by dcoutts** [comment]
bors merge

**Comment by dcoutts** [comment]
bors merge

**Pull request merged**